docker/quick-start/Dockerfile

```diff
@@ -0,0 +1,27 @@
+
+# this container should match the manual steps in 
+# https://docs.bacalhau.org/running-node/quick-start
+
+# the idea of the quickstart is to help someone bring up the simplest global cluster node
+
+# TODO: build-args for what version of bacalhau?
+
+FROM ubuntu:22.04 as bacalhau
```

Comment:
I think I'd prefer to use a proper multistage golang/scratch container here, instead of ubuntu. We're inside the repo at this point, so we can actually make nightly builds for every commit.

WDYT?

Reply:
happy to - tho I'm guessing there also needs to be a release Dockerfile that makes the tagged image from the released `bacalhau` binary

Reply:
oh, mind you, I went with ubuntu:22.04 because the docs all suggest that's what is tested - going deeper into the formal deployment images is really a matter of dev time, finding the corner cases (and worth it over time)

Reply:
Have we looked at Chainguard images? https://www.chainguard.dev/chainguard-images